cpp/src/ray/util/process_helper.cc

```diff
@@ -96,9 +96,12 @@ void ProcessHelper::RayStart(CoreWorkerOptions::TaskExecutionCallback callback)
   options.enable_logging = true;
   options.log_dir = log_dir;
   options.install_failure_signal_handler = true;
-  options.node_ip_address = "127.0.0.1";
+  std::string node_ip = ConfigInternal::Instance().node_ip_address.empty()
+                          ? "127.0.0.1"
+                          : ConfigInternal::Instance().node_ip_address;
+  options.node_ip_address = node_ip;
```

Comment:
```suggestion
  options.node_ip_address = ConfigInternal::Instance().node_ip_address;
  if (options.node_ip_address.empty()) {
    options.node_ip_address = "127.0.0.1";
  }
```

Reply:
Why?

Reply:
In this way you don't need to create a local variable and repeat `ConfigInternal::Instance().node_ip_address` twice. Just a nit.

Reply:
We need assign this value twice.
`options.node_ip_address = node_ip;` and `options.raylet_ip_address = node_ip;`